provision/filter_plugins/aws_inv.py

```diff
@@ -8,7 +8,9 @@
     from configparser import ConfigParser
 except ImportError:
     from ConfigParser import ConfigParser
-sys.path.append(os.getcwd())
+filepath = os.path.realpath(__file__)
```

Comment:
I still do not like this concept. Can we work on maybe putting in an easy_install (.pth) file or something? Eventually, I'd like to see an alternative way of doing this, but even in #ansible, they suggest something similar. 

Thinking that maybe packaging up both the modules and the filter classes into a package might work? Thoughts?